python/ray/air/_internal/torch_utils.py

```diff
@@ -56,6 +59,8 @@ def get_device() -> Union[torch.device, List[torch.device]]:
 
         devices = [torch.device(f"cuda:{device_id}") for device_id in device_ids]
         device = devices[0] if len(devices) == 1 else devices
+    elif HPU_PACKAGE_AVAILABLE and torch_hpu.is_available():
+        device = torch.device("hpu")
```

Comment:
Is it possible to allocate multiple hpu devices for a worker?

Reply:
No, HPU do not support multiple devices for single process.